org/mozilla/jss/netscape/security/x509/AlgorithmId.java

```diff
@@ -177,12 +189,88 @@ public static AlgorithmId parse(byte[] val)
     }
 
     /**
-     * Constructs a parameterless algorithm ID.
+     * Constructs an algorithm ID with a fully encoded params object
+     *
+     * @ param oid the identifier for the algorithm
```

Comment:
I think if you remove the space here, it'll make javadocs happy.

Reply:
Done.